taworks/taform/models.py

```diff
@@ -105,9 +105,15 @@ class Meta:
 class ApplicationForm(ModelForm):
     class Meta:
         model = Application
-        exclude = ('student', 'course', 'application_date', 'instructor_preference', )
-        widgets = {'reason': Textarea(attrs={'cols':50, 'rows':1}), 'preference': Textarea(
-            attrs={'cols':15,' rows':1})}
+        exclude = ('student', 'course', 'application_date', )
+    def __init__(self, *args, **kwargs):
+        super(ApplicationForm, self).__init__(*args, **kwargs)
+        self.fields['preference'].widget.attrs={
+         'class': 'pref_class'}
+        self.fields['reason'].widget.attrs={
```

Comment:
can we make it more than one row so that they can see everything they type?
![screen shot 2018-02-04 at 6 17 57 pm](https://user-images.githubusercontent.com/22120696/35783615-407042be-09d8-11e8-9197-a47a40122103.png)

Reply:
Had the same comment as @amyleblond - but know its a harder task to figure out. We should add in another spike ticket & dev ticket for this task

Reply:
<img width="918" alt="screen shot 2018-02-04 at 9 06 21 pm" src="https://user-images.githubusercontent.com/22123056/35785574-4aff645a-09ef-11e8-9f7a-647b7642cdbe.png">
I don't see a pop-up when I run your branch?

Reply:
Nevermind, didn't run collectstatic. I agree with Amy and Sarah's comments.